sr_ethercat_hand_config/demos/demo_lo.py

```diff
@@ -33,6 +33,14 @@
              'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 0.0, 'lh_THJ5': 0.0,
              'lh_WRJ1': 0.0, 'lh_WRJ2': 0.0}
 
+unpack_hand_1 = {'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 1.2, 'lh_THJ5': 0.0}
```

Comment:
If the first state is named open hand, please name this one to sth like open thumb.

Reply:
or unpack_hand_0 and unpack_hand_1